src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -67,4 +72,18 @@ object TestFormulas {
       _ <- validateNoCacheHits(metadata, workflow.testName)
     } yield ()
   }
+
+  def runFinalDirsWorkflow(wf: Workflow, dirOption: String): Test[Unit] =  {
+    def dirSize = {
+      val options = wf.data.options.get
+      val outputDir = parse(options).toOption.flatMap(_.findAllByKey(dirOption).head.asString).get
+      val dir = new java.io.File(outputDir)
+      dir.listFiles.size
```

Comment:
Maybe it's worth cleaning the outputDir. Presumably, once it succeeds, the dirSize will always be greater than 0?